.github/workflows/build.yml

```diff
@@ -86,6 +89,11 @@ jobs:
       - name: Lint
         if: ${{ steps.should.outputs.RUN == 'true' }}
         run: npm run lint:${{ matrix.service }}
+      - name: Start.sh Test 
```

Comment:
I think I need to remove this. I want to run it when 'common' changes, but maybe this is not the right spot.

Reply:
This is running it inside each service matrix; you probably just want it once, right?
Should be its own job, that `needs: service-matrix` and has a condition `if: needs.service-matrix.outputs.run-all`